GW2EIEvtcParser/ParserHelpers/Images/ParserIcons.cs

```diff
@@ -39,76 +39,112 @@ internal static class ParserIcons
     public const string UnknownProfessionIcon = "https://i.imgur.com/UbvyFSt.png";
     public const string GenericGadgetIcon = "https://render.guildwars2.com/file/09AF0498D757B191E229C862F4AA360DA65C4FE1/1012386.png";
     // High Resolution Icons 200px
+    // Ranger
+    private const string HighResGaleshot = "";
```

Comment:
Could you set the icons to unknown please? So that we have a non buggy visual until we can get the actual images, <img> does not like empty srcs